sharpy/client.py

```diff
@@ -102,11 +102,10 @@ def make_request(self, path, params=None, data=None, method=None):
             cleaned_data = data.copy()
             try:
                 del cleaned_data['subscription[ccCardCode]']
```

Comment:
FYI, you can use `cleaned_data.pop('subscription[ccCardCode]')` which will remove the value without a KeyError.